reference/6/PSReadLine/Set-PSReadlineOption.md

```diff
@@ -18,7 +18,7 @@ Customizes the behavior of command line editing in PSReadLine.
 
 ### OptionsSet
 
-```
+```powershell
```

Comment:
This is not PowerShell code it is a syntax block. We don't use the "powershell" tag on the code fence for syntax blocks. See https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/5-FORMATTING-CODE.md#formatting-illustrative-examples